docs/source/user_guide/connector/connect.ipynb

```diff
@@ -0,0 +1,106 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# ```connect()``` function"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "```connect()``` function loads the configurations and initiates the connection with the website.\n",
+    "\n",
+    "In this section, we show what parameters are available and representative examples of how to connect to different websites.\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Parameters\n",
+    "\n",
+    "* ```config_path``` is the path to the configuration file folder. There are two ways to load it. Details can be found in the previous configuraton file section.\n",
+    "\n",
+    "* ```update``` specifies if you want to pull the newest configuration files from the repo where existing configuration files are hosted.\n",
+    "\n",
+    "* ```_auth``` is the credentials to access the data passed through a dictionary. Details can be found in the authorization section.\n",
+    "\n",
+    "* ```_concurrency``` is how many queries you want to issue concurrently. By default it is one request/second. If you set it to 5, that means you want to issue five queries per second."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "\n",
+    "## Examples\n",
+    "\n",
+    "Below shows some possible ways to call ```connect()``` function.\n",
+    "\n",
+    "For how to get the API credentials, please refer to our step-by-step tutorials [here](https://github.com/sfu-db/dataprep/tree/develop/examples) for some websites.\n",
+    "\n",
+    "* **dblp** \n",
+    "\n",
+    "```\n",
+    "dblp_connector = connect('./dblp', _concurrency = 5)\n",
+    "```\n",
+    "```dblp``` is the simplest case where you do not need any credentials.\n",
+    "In this case, we load from a local configuration file folder and issue five queries per second.\n",
```

Comment:
load from a local configuration file folder -> load a local configuration file

Reply:
Here the ./dblp is a folder and except for the config file, the folder also has _meta.json for showing info() ?

Reply:
Good point!